### PR TITLE
getting the bones of chapter 1 in place

### PR DESCRIPTION
**Pull request by clm133**
Not even close to done. But I've started to get the bones in place for Lab 1:  Blink. I also added an image directory so we can better organize and reference image files.

I made a pull request to get the ball rolling on discussing how we want the projects laid out. I also am curious how the GitHub action Latex program processes pushing to a branch... 

#My initial design questions:
##'m still unsure of how much primer to put at each step? Should we explain the layout of a breadboard--power rails, terminal strips, etc--within the step of the exercise where it is relevant? Or should that be covered *before* this exercise?

##We probably want to have a common sectioning format for all projects?
Right now, I have my chapter laid out like so:
1. Overview - go over what the project entails and the end result
2. Directions - the steps to follow to get to the end result
3. Review - go into more details about what some of the direction steps entailed (i.e. trace the path of the circuit, explain the pins used, etc...)
4. Possible Expansions - how to expand on the project (add more lights, translate morse code to blinks, etc...)

### CODE REVIEW

File: micropython_workshop/project_guide/chapter_1/chapter_1.tex [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+\begin{figure}[h!]
+    \centering
+    \includegraphics[width=.6\linewidth]{chapter_1/microcontroller_seated_in_breadboard.jpg}
+    \caption{So far, so good!}
+    \label{fig:microntroller+breadboard}
+\end{figure}
+
+\subsubsection{Connect your microcontroller to the positive and negative rails of your breadboard}
+% Should we have an overview of components before this? Will the girls know what a 'jumper wire' is? Or the layout of the breadboard?
+Using a jumper wire, pin one end of the wire into hole \textbf{A1} of the breadboard and the other end on \textit{any} hole in the column labelled with a \textbf{+} on the side of the breadboard. This column is called the \textbf{positive rail}, and will eventually move power from your microcontroller around the circuit we are building.
```
Comment (clm133) [inline micropython_workshop/project_guide/chapter_1/chapter_1.tex:41]: I was following the helpful fritzing diagram as I was writing this. but just realized that connecting the 3v3 pin to the positive rail has nothing to do with the circuit we are building (at least I think its superfluous...I'm a total dummo though). Should we consider leaving this out (and update the diagram) to lessen confusion about the path of the circuit?

**Review by clm133 — COMMENTED** [review]
I'm a total dummo--no smarter than a 5th grader.

so my use of terminology  (like circuits, positive rail, negative rail, etc...) is probably, erm...wrong.

We probably should work out some consistent vocabulary and notation across the whole work book

**Comment by RobertBlackhart** [comment]
> I also am curious how the GitHub action Latex program processes pushing to a branch...

Works perfect. I didn't put any branch restrictions in the `on` condition so it will build for every push to any branch of the repo.

[Edit]: Oh, I see. Not perfect. It works as far as the build goes, but the "host it on github.io" is a little broke since it's putting everything on the same branch.

File: micropython_workshop/project_guide/chapter_1/chapter_1.tex (hunk 1 above)
Comment (RobertBlackhart) [reply micropython_workshop/project_guide/chapter_1/chapter_1.tex:41]: You're right. It's not needed in this circuit. I suppose it was just my habit. I think we should refine them to be as simple as possible, so we can remove it.

**Comment by RobertBlackhart** [comment]
> Possible Expansions - how to expand on the project (add more lights, translate morse code to blinks, etc...)

Adding morse code is a great idea! More lights is already one of the other (led_party) projects, basically.

File: micropython_workshop/project_guide/chapter_1/chapter_1.tex [hunk 2]
```diff
@@ -1,2 +1,65 @@
...
+
+\section{Directions}
+
+\subsection{Creating the circuit}
+Using jumper cables, you will be assembling a circuit between your microcontroller, your breadboard, an LED, and a 220\si{\ohm} resistor.
+
+\subsubsection{Attach the microcontroller to the breadboard}
+Carefully insert the pins at the bottom of your microcontroller into the breadboard, making sure that the microcontroller is oriented such that:
+\begin{itemize}
+    \item The pin labeled \textbf{3V3} is inserted in hole at \textbf{Column C, Row 1} of the breadboard (or \textbf{C1}, for short)
```
Comment (RobertBlackhart) [inline micropython_workshop/project_guide/chapter_1/chapter_1.tex:26]: Is it just my eyes or I would swear your bottom 3v3 pin is not seated in the board in the picture? I don't think that would even fit though so maybe it's just a camera angle?

File: micropython_workshop/project_guide/chapter_1/chapter_1.tex [hunk 3]
```diff
@@ -1,2 +1,65 @@
...
+You may need to apply more pressure than expected to seat the microcontroller properly in the breadboard. When its over, it should look like this:
+\begin{figure}[h!]
+    \centering
+    \includegraphics[width=.6\linewidth]{chapter_1/microcontroller_seated_in_breadboard.jpg}
+    \caption{So far, so good!}
+    \label{fig:microntroller+breadboard}
+\end{figure}
+
+\subsubsection{Connect your microcontroller to the positive and negative rails of your breadboard}
+% Should we have an overview of components before this? Will the girls know what a 'jumper wire' is? Or the layout of the breadboard?
```
Comment (RobertBlackhart) [inline micropython_workshop/project_guide/chapter_1/chapter_1.tex:40]: I think we should just refer to Appendix A (Electronics Essentials). Then we can just write and link to a section there.

**Comment by clm133** [comment]
> > I also am curious how the GitHub action Latex program processes pushing to a branch...
> 
> Works perfect. I didn't put any branch restrictions in the `on` condition so it will build for every push to any branch of the repo.
> 
> [Edit]: Oh, I see. Not perfect. It works as far as the build goes, but the "host it on github.io" is a little broke since it's putting everything on the same branch.

ya and I totally borked it (my .tex file ended the document early because I left the closing `\end{document}` statement in there by mistake.

File: micropython_workshop/project_guide/chapter_1/chapter_1.tex (hunk 2 above)
Comment (clm133) [reply micropython_workshop/project_guide/chapter_1/chapter_1.tex:26]: ya. that's a stinker camera angle. I'm also going to add a picture from the side of the breadboard, so they can see the depth at which the microcontroller is seated.

**Comment by RobertBlackhart** [comment]
Thanks, Charlie. This looks like a great start!

**Pull request merged**